index.html

```diff
@@ -15,6 +15,20 @@
     </style>
 </head>
 <body>
-  <h1><center>No.</center></h1>
+  <h1><center id="injury-status">Yes.</center></h1>
```

Comment:
OH LOL, I changed it to Yes to test it and forgot to change it back. WHOOPS.

Yeah, I think having the page be entirely dependent on another web resource sounds like it will never be a problem ever, so we should just assume it will always load and work as expected.